regulations/views/preamble.py

```diff
@@ -84,3 +88,49 @@ def get(self, request, *args, **kwargs):
 
         return TemplateResponse(request=request, template=template,
                                 context=context)
+
+
+class CFRChangesView(View):
+    def get(self, request, doc_number, section):
+        cfr_changes = getattr(settings, 'CFR_CHANGES', {})  # mock
+        if doc_number not in cfr_changes:
+            raise Http404("Doc # {} not found".format(doc_number))
+        versions = cfr_changes[doc_number]["versions"]
+        amendments = cfr_changes[doc_number]["amendments"]
+        label_parts = section.split('-')
+
+        if len(label_parts) == 1:
+            context = self.authorities_context(amendments, cfr_part=section)
+        else:
+            context = self.regtext_changes_context(amendments, versions,
+                                                   label_id=section)
+        return TemplateResponse(
+            request=request, template='regulations/cfr_changes.html',
+            context=context)
+
+    def authorities_context(self, amendments, cfr_part):
```

Comment:
Minor, and I'm not even sure I consider this to be a legit code smell, but these helper methods are effectively static.